app/Http/Controllers/LegacyInterOpController.php

```diff
@@ -70,6 +72,27 @@ public function indexBeatmapset($id)
         return response(null, 204);
     }
 
+    public function indexDocuments()
```

Comment:
move to its own controller in InterOp namespace

(should also look into splitting the rest as well one day™)